templates/layouts/beluga-install.hbs

```diff
@@ -0,0 +1,113 @@
+      <br>
+      <br>Enable ADB on your watch with the following steps:
+        <ol>
+          <img src="{{assets}}/img/install-beluga-1.png" class="install-centered-img"/>
+          <li>Open the settings app and go to the <b>System</b> and then <b>About</b> page.</li>
+          <li>Tap the <b>Build number</b> 7 times until Developer options become available.</li>
+
+          <img src="{{assets}}/img/install-beluga-2.png" class="install-centered-img"/>
+          <li>Go back to the settings menu and then to the <b>Developer options</b> page.</li>
+          <li>Enable <b>ADB debugging</b> and <b>OEM unlock</b>.</li>
+        </ol>
+      <br><b>Finally, to unlock the bootloader</b> enter the following commands in a terminal with your watch connected:</li>
+      <br><pre><code class="bash">adb reboot bootloader<br>fastboot oem unlock</code></pre>
+      Follow the instructions on your watch's screen. <b>Please note it may void your warranty.</b>
+      <br><br>
+      <h3>2. Flash and boot AsteroidOS</h3>
+        <div class="callout callout-warning">
+          <h4>Choose your installation type</h4>
+          <p>You can now decide if you want to replace Wear OS definitively or try AsteroidOS as a <i>dual-boot</i>. The "real installation" provides the best experience but it is advised to make a backup of your data first. The "temporary installation" is a good way to try AsteroidOS if you don't plan to use it daily.</p>
+        </div>
+
+        <ol>
+          <b><li type="a">Real installation</li></b>
+          While your watch is in bootloader mode, flash the userdata and boot partitions using Linux:
+          <pre><code class="bash">fastboot flash userdata ~/Downloads/asteroid-image-{{deviceName}}.ext4
+fastboot flash boot ~/Downloads/zImage-dtb-{{deviceName}}.fastboot
+fastboot continue</code></pre>
+
+          or using Windows:
+          <pre><code class="bash">fastboot flash userdata %systemdrive%%homepath%\Downloads\asteroid-image-{{deviceName}}.ext4
+fastboot flash boot %systemdrive%%homepath%\Downloads\zImage-dtb-{{deviceName}}.fastboot
+fastboot continue</code></pre>
+
+          <b><li type="a">Temporary installation</li></b>
+          <b>Important to not boot into WearOS after unlock your bootloader!</b>
+          While your watch is in bootloader mode, flash the recovery partitions using Linux:
+          <pre><code>fastboot flash recovery ~/Downloads/zImage-dtb-{{deviceName}}.fastboot</code></pre>
+
+          or on Windows:
+          <pre><code>fastboot flash recovery %systemdrive%%homepath%\Downloads\zImage-dtb-{{deviceName}}.fastboot</code></pre>
+          Then, Boot the temporary image (using the on screen boot menu). The bootlogo will popup with the message "<b>Push asteroidos.ext4 via ADB</b>" will show below the bootlogo.
```

Comment:
It isn't clear how one would boot into the temporary image.
Maybe add here that one has to select recovery mode and boot to that:
While in recovery mode use the top button on the watch to select "Recovery mode" and push the bottom button to boot into the temporary image of AsteroidOS.